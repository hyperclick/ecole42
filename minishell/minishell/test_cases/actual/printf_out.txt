ptr: 3231760
number: 1, char: a, string: 'abc'
: 1
str1_str2
1
str
