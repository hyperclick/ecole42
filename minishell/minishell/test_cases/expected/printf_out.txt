ptr: 0x10000a128
number: 1, char: a, string: 'abc'
: 1
str1_str2
1
str
